tests/plenary/state/state_spec.lua

```diff
@@ -80,9 +80,11 @@ describe('State', function()
     state:save_sync()
 
     -- Mangle the cache
-    vim.cmd.edit(cache_path)
-    vim.api.nvim_buf_set_lines(0, 0, -1, false, { '[ invalid json!' })
-    vim.cmd.write()
+    vim.schedule_wrap(function()
+      vim.cmd.edit(cache_path)
+      vim.api.nvim_buf_set_lines(0, 0, -1, false, { '[ invalid json!' })
+      vim.cmd.write()
+    end)
```

Comment:
Can you try these changes instead and see if it works?

```suggestion

    vim.cmd(('edit %s'):format(cache_path))
    vim.api.nvim_buf_set_lines(0, 0, -1, false, { '[ invalid json!' })
    vim.cmd('write')

```

From the screenshot I see that neotest-plenary cannot figure out that `vim.cmd` is a table, so this might fix it.

Reply:
Thanks, this actually also fixes the problem. I updated my commit with your suggestion.